bigchaindb/commands/bigchaindb.py

```diff
@@ -158,6 +159,40 @@ def run_election_new_upsert_validator(args, bigchain):
         return False
 
 
+def run_election_new_migration(args, bigchain):
+    """Initiates an election to halt block production
+
+    :param args: dict
+        args = {
+        'sk': the path to the private key of the node calling the election (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: election_id or `False` in case of failure
+    """
+
+    try:
+        key = load_node_key(args.sk)
+        voters = MigrationElection.recipients(bigchain)
+        election = MigrationElection.generate([key.public_key],
+                                              voters,
+                                              {}, None).sign([key.private_key])
+        election.validate(bigchain)
+    except ValidationError as e:
+        logger.error(e)
+        return False
+    except FileNotFoundError as fd_404:
+        logger.error(fd_404)
+        return False
```

Comment:
I think the above logic is same for `election new upsert-validator`? It would be good to abstract